internal/telemetry/cluster.go

```diff
@@ -145,12 +148,37 @@ func (c *Collector) IngressClassCount(ctx context.Context) (int, error) {
 	return len(ic.Items), nil
 }
 
-// PolicyCount returns number of Policies watched by NIC.
-func (c *Collector) PolicyCount() int {
+// PolicyCount returns the count of fields in each PolicySpec
+func (c *Collector) PolicyCount() v1.PolicySpecMap {
+	policyCounters := make(v1.PolicySpecMap)
 	if c.Config.Policies == nil {
-		return 0
+		return map[string]int{}
```

Comment:
return types (?) L155, L160

Reply:
I had added this line to prevent a nil pointer error. I now have realised they can both be map[string]int and there is no need for a custom type.